### PR TITLE
Filter out the invalid POST request

### PR DESCRIPTION
**Pull request by anakin1028**
We only need to decode with the POST request which passes the phishing information. 

fix for #562

### CODE REVIEW

File: wifiphisher/common/phishinghttp.py [hunk 1]
```diff
@@ -57,19 +57,26 @@ def post(self):
...
 
         global terminate
-        post_data = tornado.escape.url_unescape(self.request.body)
 
-        # log the data
-        log_file_path = "/tmp/wifiphisher-webserver.tmp"
-        with open(log_file_path, "a+") as log_file:
-            log_file.write("[{0}*{1}] {2}POST{1} request from {0}{3}{1} with {0}{4}{1}\n".format(
-                T, W, O, self.request.remote_ip, post_data))
+        valid_content_type = "application/x-www-form-urlencoded"
```
Comment (sophron) [inline wifiphisher/common/phishinghttp.py]: Can you move this on `constants.py`?

**Comment by anakin1028** [comment]
@sophron thanks : )

**Comment by blackHatMonkey** [comment]
@anakin1028 I will test it some time this week 😄 .

**Comment by sophron** [comment]
Looks good. @blackHatMonkey can you proceed with testing/merging?

**Comment by blackHatMonkey** [comment]
I will test this today and merge it 😄.

**Comment by blackHatMonkey** [comment]
@anakin1028 This looks good. One thing I noticed not because of this PR but for each POST request we are adding `/requested page` to the end of the message. Do you have any idea where it is being added?

**Comment by anakin1028** [comment]
@blackHatMonkey I didn't notice about that. Do you mean the `body` from the `tornado's` post request are different from the POST request on the capture?

**Comment by blackHatMonkey** [comment]
@anakin1028 So to give you an example if the requested address is `http://clients3.google.com/generate_204/generate_204` and the password I inputted is `abc` then it would show it like so:
`wfphsr-password=abc/generate_204`.

**Pull request merged**